src/Symfony/Component/Cache/Adapter/RedisAdapter.php

```diff
@@ -36,13 +35,15 @@ public function __construct(\Redis $redisConnection, $namespace = '', $defaultLi
      */
     protected function doFetch(array $ids)
     {
-        $values = $this->redis->mget($ids);
-        $index = 0;
-        $result = [];
-
-        foreach ($ids as $id) {
-            if (false !== $value = $values[$index++]) {
-                $result[$id] = unserialize($value);
+        $result = array();
```

Comment:
@Nek- our policy is to not use them, to keep consistency in the codebase (and we cannot convert the whole codebase to the short array syntax, as we will need to merge 2.8 into 3.x during 3 years, and we don't want git conflicts each time we change a line dealing with an array)